users/views.py

```diff
@@ -1,3 +1,13 @@
-# from django.shortcuts import render
+from django.shortcuts import render
 
-# Create your views here.
+
+def profile(request):
+    return render(request, 'users/profile.html', {'title': 'profile'})
+
+
+def login(request):
+    return render(request, 'users/login.html', {'title': 'login'})
+
+
+def signin(request):
+    return render(request, 'users/signin.html', {'title': 'signin'})
```

Comment:
Same here

Reply:
> Login is for registered users, and sign-in is for new users that want to register.

I think we should not mess with registration, Barak told us not try implementing it, it may be complicated.
We have the 5 hardcoded users\profile in the DB, it should be enough in my opinion.